tracing-core/src/metadata.rs

```diff
@@ -283,6 +283,19 @@ impl Level {
     ///
     /// Designates very low priority, often extremely verbose, information.
     pub const TRACE: Level = Level(LevelInner::Trace);
+
+    /// Get the string representation of the `Level`.
+    ///
+    /// This returns the same string as the `fmt::Display` implementation.
+    pub fn as_str(&self) -> &str {
```

Comment:
should this return an `&'static str`? i don't think there's really any reason for it to ever need to return a string that isn't valid for the `'static` lifetime, so it may as well be?

```suggestion
    pub fn as_str(&self) -> &'static str {
```